### DISCUSSION TITLE
Fulfilment Manual Testing

### DISCUSSION THREAD

**Discussion by KirinZero0** [inikoo/aiku#589] [General]
1. [x] 1 Edit Cut Off Day (Week and Month) 31 and Wednesday 
2. [x] 2 Create Fulfilment Customer 
3. [x] 3 Edit Fulfilment Customer 
4. [x] 4 Create WebUser 
5. [x] 5 Edit WebUser 
6. [x] 6 Login in Retina 
7. [x] 7 Forgot Password (Email method)
8. [x] 8 Forgot Password (Change from aiku)
9. [x] 9 Create Rental Agreement (Weekly) 
10. [x] 10 Create Another Customer 
11. [x] 11 Create Rental Agreement (Monthly) 
12. [x] 12 Create Pallet Delivery for both and Set it to booked in 
13. [x] 13 Check if the Recurring Bill is Created 
14. [x] 14 Check the Weekly end date (Weekly Customer) 
15. [x] 15 Check the Monthly end date (Monthly Customer
16. [x] 16 go to setting and change the cut off day again 11 and Tuesday
17. [x] 17 Check the end date again

**Comment by KirinZero0** [comment]
@aqordeon @YudhistiraA number 2, i dont think the phone number field work properly can you please check on it?

**Reply by inikoo** [reply]
Issue created #611

**Reply by inikoo** [reply]
Now there is other validation , let see if this works

**Comment by KirinZero0** [comment]
![image](https://github.com/user-attachments/assets/828a0519-c7fe-4038-bc7e-c2f798f59504)

Retina reset password via email is failing

@inikoo @itzArtha

**Reply by inikoo** [reply]
issue #612

**Comment by KirinZero0** [comment]
**SOLVED** ✅

![image](https://github.com/user-attachments/assets/f0d49085-49fb-4e02-b15d-9641645a0b24)

![image](https://github.com/user-attachments/assets/99d4b458-0e26-4789-b7fe-d8322a99b037)

![image](https://github.com/user-attachments/assets/ae292d64-9a1d-4b7f-adbf-9cca6e64fd84)

![image](https://github.com/user-attachments/assets/5618c019-7c50-4c57-8815-31d7c669bf09)

![image](https://github.com/user-attachments/assets/a0b948d7-4756-4359-92a1-7aca8476223c)


In Retina show Pallet Delivery, fails to fetch service and physical goods data and service data isnt shown properly in the table (the physical goods probably too), There's also and error if i add physical goods, it throws a console error in retina whenever i switch to physical goods tab.

Theres also an error where currency id is null (weird)

but its normal in aiku

@inikoo @aqordeon @YudhistiraA @itzArtha

**Comment by YudhistiraA** [comment]
if the search is empty it will redirect to a page that must create a customer @aqordeon 
![image](https://github.com/user-attachments/assets/4528181b-5820-49ad-be03-f7a95f67783f)

**Reply by inikoo** [reply]
This stuff related to delayed hydrators, so is ok (maybe)

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/5e2b0a53-996f-4b0d-82d8-de646e9d2b79)
cannot setting websites fullfilment in stanging @inikoo @aqordeon @YudhistiraA @itzArtha

**Reply by inikoo** [reply]
Issue created #613

**Comment by YudhistiraA** [comment]
**Solved** ✅

![image](https://github.com/user-attachments/assets/3bf97416-2f0b-4d01-92a7-21d67104847f)
it's better not to have a status filter on pallet delivery @aqordeon @itzArtha

**Reply by aqordeon** [reply]
@itzArtha @KirinZero0

**Comment by YudhistiraA** [comment]
**SOLVED NO NEED**

![image](https://github.com/user-attachments/assets/b2b37b8c-738a-4b99-a09f-9e5b88773421)
![image](https://github.com/user-attachments/assets/bc7c16ca-9d64-4267-8d41-ee84c1eda3a7)
user doesn't have a rental agreement on aiku, user still can make delivery on Retina

@itzArtha @YudhistiraA @aqordeon

**Reply by inikoo** [reply]
No need because they should not be able to be here if no rental agreement #599

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/b85cccf0-cd5d-40af-9198-2450e40f78fc)
in retina doesn't have notes @aqordeon

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/ca10674d-12f3-4b9d-aa8d-a26f5cf8e518)
retina must have export to pdf too

**Reply by inikoo** [reply]
issue cereated #614

**Comment by YudhistiraA** [comment]
**DONE**

![image](https://github.com/user-attachments/assets/5a23f3de-cf1c-4d29-9a64-0b31fa703edd)
services has have different tables @aqordeon

**Reply by aqordeon** [reply]
fixed in locally, wait for it in staging

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/1fdb0194-73f6-4073-94e6-a2e9ebb0d8d6)
dont have pallet return in retina

**Comment by YudhistiraA** [comment]
**SOLVED**

![image](https://github.com/user-attachments/assets/873c2255-38b2-49fd-85ae-787384bdd5b9)
cannot go to pallet delivery in retina

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/1e8077db-d0e1-461a-9421-2934551261f2)
need state in the column and not just icon in the type of palllet @aqordeon

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/2b53f276-6e1a-4ff5-ba24-5de8ca9a4f4a)
delete action Colum in retina

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/6147d83c-a267-4f1a-bda8-4f3c64b25b8d)
serach not working @KirinZero0

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/ad1d5fc3-eb25-481c-afd5-b25b31d2ed92)
Better retina has that part @aqordeon @YudhistiraA @inikoo @itzArtha

**Reply by aqordeon** [reply]
issue #610

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/f0eb642b-0b94-45b5-b344-f1a9fe9f0bcd)
add Physical Goods in retina not working @KirinZero0

**Comment by YudhistiraA** [comment]
![image](https://github.com/user-attachments/assets/a1198e29-a4af-42e4-b14c-5dbaa4ff489b)
add services in retina not work

**Comment by KirinZero0** [comment]
![image](https://github.com/user-attachments/assets/8dffa3b6-4646-41c1-b7eb-40bdf1e719b6)


Error when creating PalletDelivery

@inikoo @itzArtha
